File: PatternAPI/PatternAPI/Models/ValueDomainValue.cs
namespace PatternsAPI.Models
{
    public class ValueDomainValue : BaseEntity
    {
        public string? Value { get; set; }
        public int ValueDomainId { get; set; }
        public ValueDomain? ValueDomain { get; set; }
        public int EntityId { get; set; }


    }
}

File: PatternFrontEnd/patterns-manager/src/app/patterns/_models/value-domain-value.ts
import { Base } from "src/app/shared/_models/base";
import { ValueDomain } from './../../settings/_models/value-domain';

export interface ValueDomainValue extends Base {
  value?: string;
  valueDomainId?: number;
  valueDomain?: ValueDomain;
  entityId?: number;
}
